packages/terra-worklist-data-grid/src/proptypes/cellShape.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types';
+
+const cellShape = PropTypes.shape({
+  /**
+   * String indicating content to be displayed for the cell.
+   */
+  text: PropTypes.string,
```

Comment:
Your shape does not match your example JSON data.  In the example JSON, this is called cellContent.

Reply:
Great catch, modified.